### PR TITLE
Add ed25519 2020 context.

### PR DESCRIPTION
**Pull request by msporny**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # bedrock-ledger-context ChangeLog
 
+## 18.1.0 - 2021-xx-xx
```
Comment (dlongley) [inline CHANGELOG.md]: We always do major released when contexts change.

```suggestion
## 19.0.0 - 2021-xx-xx
```

**Review by dlongley — APPROVED** [review]
Approving -- we need to review if this pattern (bundling contexts together in a single module) fits in with our move to reference contexts on an individual basis and import them where needed. The latter has shown promise for resolving a number of issues, but it could be that for certain bedrock applications some bundling still makes sense -- I don't know.

**Comment by msporny** [comment]
> we need to review if this pattern (bundling contexts together in a single module) fits in with our move to reference contexts on an individual basis and import them where needed.

My current opinion is that the pattern is broken and we need to figure out something else to do:

https://github.com/digitalbazaar/bedrock-ledger-context/blob/v1-dingo-rc1/lib/index.js#L29-L50
https://github.com/veres-one/did-veres-one/blob/v14.x/lib/documentLoader.js#L13-L20

That feels unsustainable... we use /both/ of those document loaders in parallel in veres-one-validator, btw... which gets super confusing when they depend on different context versions. :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # bedrock-ledger-context ChangeLog
 
+## 19.0.0 - 2021-xx-xx
+
+### Added
+- Add ed25519-signature-2020-context@1.
```
Comment (msporny) [inline CHANGELOG.md:6]:
```suggestion
- Add ed25519-signature-2020-context@1.
- Add x25519-key-agreement-2020-context@1.
```

**Pull request merged**